sources/packages/backend/apps/db-migrations/src/sql/Reports/Create-data-inventory-report.sql

```diff
@@ -0,0 +1,505 @@
+                    disbursement_overawards.student_id,
+                    disbursement_overawards.disbursement_value_code
+            ) as "BCSL Overawards Balance",
+            students.disability_status as "Permanent Disability status",
+            -- Application Personal Info (does not include Institution or Program info)
+            ''"'' || program_years.program_year_desc || ''"'' as "Program Year",
+            applications.data ->> ''studentNumber'' as "Student Number",
+            applications.data ->> ''citizenship'' as "Citizenship status",
+            applications.data ->> ''bcResident'' as "BC Residency Status",
+            applications.data ->> ''indigenousStatus'' as "Indigenous Person status",
+            applications.data ->> ''aboriginalHeritage'' as "Indigenous category code",
+            applications.data ->> ''applicationPDPPDStatus'' as "PD/PPD Application Flag",
+            applications.data ->> ''youthInCare'' as "Youth in Care Flag",
+            ''--TBD--'' as "Youth in Care beyond age 19 Flag",
+            applications.data ->> ''outOfHighSchoolFor4Years'' as "Outside of highschool >4 years flag",
+            applications.data ->> ''whenDidYouGraduateOrLeaveHighSchool'' as "High School Graduation Date",
+            applications.data ->> ''fulltimelabourForce'' as "Recent Labour Force Participation Flag",
+            applications.data ->> ''addTrustContactToContactSABC'' as "Trusted Contact Identified (Yes/No)",
+            applications.data ->> ''relationshipStatus'' as "Marital Status",
+            --Application Financial Info
+            applications.data ->> ''taxReturnIncome'' as "Total income",
+            applications.data ->> ''partTimeAwardTypesToBeConsidered'' as "Grant-only flag",
+            applications.data ->> ''scholarshipsreceviedCosts'' as "Merit-based scholarship amount",
+            applications.data ->> ''parentvoluntarycontributionsCosts'' as "Voluntary parental contribution amount",
+            applications.data ->> ''governmentFundingCosts'' as "Government funding amount",
+            applications.data ->> ''nongovernmentfundingCosts'' as "Non-government funding amount",
+            applications.data ->> ''bcIncomeassistancefordisabilitiesCosts'' as "B.C. Income Assistance Amount",
+            applications.data ->> ''returntriphomeCost'' as "Relocation costs",
+            -- Application Financial Appeals Info
+            applications.data ->> ''exceptionalExpensesApplicationException'' as "Exceptional expense amount",
+            applications.data ->> ''daycareCosts11YearsOrUnder'' as "Daycare cost amount",
+            applications.data ->> ''totalUnsubsidizedDayCareCostsDuringClassHoursForTheProposedStudyPeriodForDisabledInfirmedDependents12YearsOfAgeAndOlderPanel'' as "Unsubsidized daycare amount",
+            ''--TBD--'' as "Child/Spousal Support amount",
+            ''--TBD--'' as "Living in parent house and paying living expenses amount",
```

Comment:
I appreciate that you added a placeholder column for the data which is not part of the ticket. 
Please make sure that it is mentioned in the ticket and also business is aware of it.

Reply:
Yes, it is already mentioned: 
![image](https://github.com/bcgov/SIMS/assets/78114138/e709c624-4a23-48e9-b96a-7d10d9b1c418)

And I also sent a sample to the business people and they were communicated about it.

Reply:
I meant the ticket not the PR. 

And I also sent a sample to the business people and they were communicated about it. - Great 👍